products/compute/terraform.yaml

```diff
@@ -2430,11 +2430,18 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   TargetInstance: !ruby/object:Overrides::Terraform::ResourceOverride
     examples:
       - !ruby/object:Provider::Terraform::Examples
+        min_version: beta
```

Comment:
Oops, I think you added `min_version` to the wrong example :)

Reply:
😃  fixed